src/displayapp/screens/StopWatch.cpp

```diff
@@ -123,97 +143,113 @@ void StopWatch::SetInterfaceStopped() {
   lv_obj_set_state(txtStopLap, LV_STATE_DISABLED);
 }
 
-void StopWatch::Reset() {
-  SetInterfaceStopped();
-  currentState = States::Init;
-  oldTimeElapsed = 0;
-  lapsDone = 0;
+void StopWatch::RenderTime() {
+  TimeSeparated elapsedTime = ConvertTicksToTimeSegments(stopWatchController.GetElapsedTime());
+  SetHoursVisible(elapsedTime.hours != 0);
+  if (!hoursVisible) {
+    lv_label_set_text_fmt(time, "%02d:%02d", elapsedTime.mins, elapsedTime.secs);
+  } else {
+    lv_label_set_text_fmt(time, "%02d:%02d:%02d", elapsedTime.hours, elapsedTime.mins, elapsedTime.secs);
+  }
+  lv_label_set_text_fmt(msecTime, "%02d", elapsedTime.hundredths);
```

Comment:
Out of scope for this PR probably, but for future reference DirtyValue (just pass in seconds) could be used to avoid redrawing all of the digits every frame (so the hundreths/tenths update faster)

Reply:
Thanks for pointing this out. I'd somehow thought the widget was smart enough to avoid redundant rendering but that's clearly not the case. So yeah, it's probably out of scope but I applied it anyway. Saving cycles in the refresh task can't hurt.